### ISSUE TITLE
Network weights

### ISSUE THREAD

**Issue by genbing99** [Necolizer/Facial-Prior-Based-FOMM#1]
Hi, can you share the checkpoint zip during the network training to achieve the final results in your paper?

**Comment by Necolizer** [comment]
> Hi, can you share the checkpoint zip during the network training to achieve the final results in your paper?

Yep, here's the Google Drive share link: _[Here used to be a share link but we found bug in that file. See below.]_
It was trained on the composite dataset of CASME2, SAMM and SMIC-HS. Hope this might be helpful for you.

**Comment by genbing99** [comment]
Thank you for sharing.
I'm getting error when running the line in logger.py:
`kp_detector.load_state_dict(checkpoint['kp_detector'])`
The error showing:
`Error(s) in loading state_dict for KPDetector: Unexpected key(s) in state_dict: "conv1.weight", "conv1.bias", "conv2.weight", "conv2.bias". `
Please advise for the error.

Besides, may I know where can I get the pictures of westernMale and asianFemale for evaluation?

**Comment by Necolizer** [comment]
@genbing99 Source videos and target faces for evaluation can be found at https://megc2021.github.io/images/MEGC2021_generation.zip , which is also provided at https://megc2021.github.io/agenda.html , in section **Description of Challenge tasks**.
The error seems so strange. I will @ you later after a day or two if the solution is found, since now i am busy for my mid-term homework and exams. Sorry for that. QAQ

**Comment by genbing99** [comment]
Sure, thanks in advance.

**Comment by Necolizer** [comment]
@genbing99 Hey, pls try loading this new checkpoint file to check if it works: https://drive.google.com/file/d/15U531P0h2ujql_ag2-mht2tjUlJFYZ_H/view?usp=sharing

The error is caused by redundant code in `modules/keypoint_detector.py`, which have already been deleted when pushing to this Github repository. But we did not notice that the redundant code were still there when the training started.
```
class KPDetector(nn.Module):
    def __init__(...):
        // ...
        self.conv1 = torch.nn.Conv2d(...)// This line
        self.conv2 = torch.nn.Conv2d(...)// And this line
```
Two convolution layers `self.conv1` & `self.conv2` were defined in `class KPDetector`, but never used during training or animating. They were initialized and saved into the checkpoint. The logger could not recognize the initialized parameters of these two redundant layers and then threw out the error, since the code you pulled from this repository has been corrected.
 
So the solution is to delete the key-value pairs (key == "conv1.weight" or "conv1.bias" or "conv2.weight" or "conv2.bias") in "kp_detector" part of the checkpoint file. I did it and it works on my server. The generation results are the same as the ones we obtained in our paper. The new checkpoint file is shared via the new share link above. Btw, the previous Google Drive share link has been cancelled.

Hope this might be helpful for you.
